React-Redux Bookstore - Fifth Milestone

Rhelli opened the pull request:
# 📚 React-Redux BookStore - Milestone 5 - Zeplin & Styling

Hi all,

We have finished the fifth milestone of the React & Redux Bookstore project and have completed the following steps:

 - Implemented the styling as per the Zeplin styling brief
 - Made sure that the styling matches the brief as closely as possibly
 - Pushed the new styled version of the app to Heroku (the link can be found in the ReadMe)

Thanks

Alexis and Rory

Rhelli committed:
Began Adding Styling Components and Styling The Project:

 - Added in each component/containers css module
 - Installed node-sass for scss compatibility
 - Added styling for index, top title bar and background of app container

Psiale committed:
Obliterated the table to instead use divs and flex to style

Psiale committed:
Styled books components according to the provided style specifications

Rhelli committed:
Finished off adding last CSS styling modules. Styled the bottom form and positioned everything. Enabled scroll on the booklist to prevent it overflowing its container when more books are added.

trekab reviewed (changes requested):
### Changes Required! :recycle: 
Hi team! Your current effort on this milestone is commendable. However, there are some details I'd like you to work on before moving forward:
- [x] The header section with the `Bookstore CMS` title, `Books`, `Categories` and the `user icon` elements is missing from your project.
- [x] The books displayed should show the `category`, `title`, and `author` details for the specific book. Please maintain the same order and structure given in the design.
- [x] Please note that instead of the `delete` button, the design specifies the `Comments`, `Remove`, and `Edit` options for each displayed book. 
- [x] Instead of `Submit` on the button creating a new book, I'd suggest you stick to the design given and use `ADD BOOK`.
- [ ] Write your commits in the imperative/present tense ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_commits.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) This [video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) is a great guide to editing commit messages using `git rebase`

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

Psiale committed:
Fixing styling requirements

Rhelli committed:
This commit fixes:

 - Adds missing profile/login button in header container
 - Adds missing comments, author buttons
 - Renames the 'delete' button to 'remove book' in book component
 - Renames the 'submit' button in the BookForm component to 'Add Book'

Rhelli commented:
# 📚 React-Redux Bookstore - Milestone 5 - Pull #2 

Hi there,

Following the requested changes by the TSE above, we have made the following changes / fixes to the project:
 - Added the missing login / profile button from the site header
 - Added the author and comments buttons on the listed book items themselves
 - Renamed the 'Delete' button to 'Remove Book' on the listed book items
 - Renamed the 'Submit' button in the BookForm component (at the bottom of the page) to 'Add Book'


### Note 

We're not sure what the first suggestion was asking for, as the header is (and was) present on the project originally - it's most likely due to the weirdly large specified dimensions provided from Zeplin that it may not be visible. It is, however, there and coded in validly.

Thanks,

Alexis and Rory

Torres-ssf reviewed (approved):
### Milestone approved :+1: 
Hi Team :hand: 
Great job implementing the requested changes.
Good luck with the next milestone

Happy coding :computer: 
**Sergio Torres**
